src/ws-option-buttons/ws-option-buttons.js

```diff
@@ -158,18 +157,18 @@ export class WSOptionButtons extends Component {
   render() {
     return (
       <div className="ws-option-buttons" ref={element => { this.element = element; }}>
-        {this.state.items.map((item, index) =>
+        {this.state.items.map((item, index) => (
           <div className={`option-button ${index < this.state.visible ? '' : 'is-hidden'}`}>
-            <a
+            <a // eslint-disable-line jsx-a11y/anchor-is-valid
```

Comment:
I would not just use rules because others say that they are right and useful. When they don't apply to our code we don't need to disable them on every error manually but can disable it completely